scripts/skills/perks/perk_rf_decisive.nut

```diff
@@ -0,0 +1,160 @@
+this.perk_rf_decisive <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		FatigueMult = 0.85,
+		ResolveModifier = 15,
+		DamageMult = 1.15,
+		AttractionMult = 1.1,
+
+		// Const
+		Stacks = 0,
+		MaxStacks = 3,
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_decisive";
+		this.m.Name = ::Const.Strings.PerkName.RF_Decisive;
+		this.m.Description = "This character is capable of making quick and confident decisions without any hesitation.";
+		this.m.Icon = "ui/perks/rf_decisive.png";	// TODO implement
+		this.m.IconMini = "rf_decisive_mini";	// TODO implement
+		this.m.Type = ::Const.SkillType.Perk | ::Const.SkillType.StatusEffect;
+		this.m.Order = ::Const.SkillOrder.Perk;
+		this.m.IsActive = false;
+		this.m.IsStacking = false;
+		this.m.IsHidden = true;
+	}
+
+	function getName()
+	{
+		return (this.m.Stacks == 0) ? this.m.Name : this.m.Name + " (x" + this.m.Stacks + ")";
```

Comment:
It's a personal choice for readability.
If you don't want brackets used in such cases as per coders choice, then I'd like the opinion of the rest tech team and a definition on if/when they are to be used for readability and when - not.

Reply:
This is still unresolved and the new ternaries you've added have the same issue. It is rather unorthodox to use parantheses in ternary statements like this. We already talked about this in Discord - so please remove them. I'd expect parantheses to be for some reason and looking for that reason consumes time. See example below. When I see a paranthesis opening I subconsciously expect a reason for it and that consumes effort and time when they are used without reason.
```squirrel
local myVar = (someBool == true ? val1 : val2) * 2;
local myVar = someBool == true ? val1 : val2 * 2;
```
The above code shows a good case of using parantheses because the order of operations changes with them and `myVar` ends up being different. Although the first line I'd personally write as 
```squirrel
local myVar = 2 * (someBool == true ? val1: val2);
```
but that's besides the point.

If you are having issues with readability without the parantheses it may be because of your code editor's color coding, perhaps? Because to me the version without parantheses looks quite readable. It's also a case of being used to a particular style I guess.

Reply:
If you think this is not a matter of personal preference but instead always must be done a certain way then you should formulate that as a hard/soft rule and we can discuss over at discord whether to add that to the SOPs.